cromshell

```diff
@@ -608,20 +704,86 @@ function fetch-all()
 # When it is, send an email to the email address specified.
 function notify()
 {
+  if [[ $# -lt 1 ]] ; then
+    error "Error: You must specify an email address to notify."
+    error ""
+    simpleUsage notify
+    echo "Try \`$SCRIPTNAME -h' for more information."
+    exit 5
+  fi
 
+  # Is the next argument our workflow ID?
+  # Workflow IDs are standard UUIDs or negative numbers:
+  if [[ $(matchWorkflowId ${1}) -eq 0 ]] || [[ ${1} == -* ]] ; then
```

Comment:
does `[[ ${1} == -* ]]` read as: " the value of $1 is equal to a `-` followed by any characters" ?  If so should it be a match of `-[:digit:]+` instead?